app/src/main/java/com/noblemajesty/brt/database/entities/BusSchedule.kt

```diff
@@ -13,12 +13,13 @@ data class BusSchedule (
         var destination: String,
         var from: String,
         var status: String,
-        var date: String
+        var date: String,
+        var cost: String
 )
 {
     @Ignore
-    constructor(): this(0, 0, "", "", "", "", "")
+    constructor(): this(0, 0, "", "", "", "", "", "")
```

Comment:
Since you are using Room database, writing this constructor might be redundant since you can initialize the values. Good work setting this up